presto-tests/src/main/java/io/prestosql/tests/AbstractTestAggregations.java

```diff
@@ -847,8 +847,8 @@ public void testGroupByNanArray()
         List<MaterializedRow> actualRows = actual.getMaterializedRows();
         assertEquals(actualRows.size(), 1);
         assertTrue(Double.isNaN(((List<Double>) actualRows.get(0).getField(0)).get(0)));
-        assertEquals(((List<Double>) actualRows.get(0).getField(0)).get(1), 2.0);
-        assertEquals(((List<Double>) actualRows.get(0).getField(0)).get(2), 3.0);
+        assertEquals((double) ((List<Double>) actualRows.get(0).getField(0)).get(1), 2.0);
+        assertEquals((double) ((List<Double>) actualRows.get(0).getField(0)).get(2), 3.0);
```

Comment:
Why is this needed?
(do we needed this elsewhere?)

Reply:
It looks like that they changed API and otherwise it fails the compilation. I compiled all project and no other changes were needed.